ui/apps/platform/src/Containers/Vulnerabilities/components/SnoozedCveToggleButton.cy.jsx

```diff
@@ -0,0 +1,61 @@
+import React, { useState } from 'react';
+import SnoozeCveToggleButton from './SnoozedCveToggleButton';
+
+function Wrapper({ startingSearchFilter = {} }) {
+    const [searchFilter, setSearchFilter] = useState(startingSearchFilter);
+
+    return (
+        <>
+            <div>
+                <h1>Filters</h1>
+                {Object.entries(searchFilter).map(([key, values]) => (
+                    <div key={key}>
+                        {key}:{values.join(',')}
+                    </div>
+                ))}
+            </div>
+            <SnoozeCveToggleButton searchFilter={searchFilter} setSearchFilter={setSearchFilter} />
+        </>
+    );
```

Comment:
Since the functionality of this component is primarily used to drive state and setState functions that are passed as props, I'm using a simple `useState` hook to render the state to the DOM to make testing more straightforward.

This also lets us manage the implementation details of how the CVE Snooze filter is applied, such as avoiding the browser's URL in this component level test.